Resources/Prototypes/_Scp/Entities/Mobs/Player/Scp/scp.yml

```
@@ -1,9 +1,26 @@
 ﻿# Пресеты для артефактов-сцп.
 # Для разных сцп нужны разные пресеты
 
+- type: entity
+  id: BaseScpArtifact
+  abstract: true
+  components:
+  - type: ContainerContainer
```

YAMLlint (1.37.1)

[error] wrong indentation: expected at least 3

(indentation)